compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/DatePicker.kt

```diff
@@ -735,7 +736,7 @@ class DatePickerColors internal constructor(
  */
 @ExperimentalMaterial3Api
 @Immutable
-class DatePickerFormatter constructor(
+class DatePickerFormatter(
```

Comment:
Let's postpone the merge, until we merge 1.2.

Otherwise it will be difficult to merge